### PR TITLE
[microNPU] modify the demo to use USMP

### PR DESCRIPTION
**Pull request by manupa-arm**
This commit changes the demo to use USMP.

### CODE REVIEW

**Comment by manupa-arm** [comment]
cc : @Mousius @grant-arm

File: apps/microtvm/ethosu/run_demo.sh [hunk 1]
```diff
@@ -151,6 +151,10 @@ python3 -m tvm.driver.tvmc compile --target=ethos-u,cmsis-nn,c \
     --executor=aot \
     --executor-aot-interface-api=c \
     --executor-aot-unpacked-api=1 \
+    --executor-aot-unpacked-api=1 \
```
Comment (grant-arm) [inline apps/microtvm/ethosu/run_demo.sh:154]: This line is duplicated.

**Comment by grant-arm** [comment]
Thanks for this @manupa-arm ! This is fantastic.
Just one nit, otherwise it LGTM!

**Comment by manupa-arm** [comment]
Thanks for the quick feedback! @grant-arm . I did the change now.

**Comment by grant-arm** [comment]
LGTM!

**Pull request merged**